ultimaker_pla_175.xml.fdm_material

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+    <metadata>
+        <name>
+            <brand>Ultimaker</brand>
+            <material>PLA</material>
+            <color>Generic</color>
+        </name>
+        <GUID>abb9c58e-1f56-48d1-bd8f-055fde3a5b56</GUID>
+        <reference_material_id>pla</reference_material_id>
+        <version>25</version>
+        <color_code>#f1ece1</color_code>
+        <description>Fast, safe and reliable printing. PLA is ideal for the fast and reliable printing of parts and prototypes with a great surface quality.</description>
+        <adhesion_info>Print on bare glass. Use tape for cold build plates.</adhesion_info>
```

Comment:
If i'm correct, this material is only for Method printers, so this advice is quite inappropriate...

Reply:
Yes! good catch.  I just reached out to a few people about this message and will push a commit soon.  I will also update the PVA messages.